rhcos4/profiles/moderate.profile

```diff
@@ -1,5 +1,10 @@
 documentation_complete: true
 
+metadata:
+    reference: https://nvd.nist.gov/800-53/Rev4/impact/moderate
```

Comment:
One more thing. I would like `reference` promoted to a profile element and built as part of the build system. This is useful to upstream and downstream.

Reply:
I can give it a try, but if it get complicated I'd rather do it in subsequent PR.

Reply:
With 1a332a4 the metadata reference is exposed in the Profile XML as a reference.

Reply:
Thanks @yuumasato, but `references` should no longer be under metadata. This should be analogous now to how references are done in the rule.yml files.

Reply:
Ok.
Just a note that the references are put as simple strings.

Reply:
Okay... makes sense for now. I imagine a list of reference strings will need to be handled, but we can deal with that later.